main.go

```diff
@@ -63,6 +89,7 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	// Create a new manager to manage the operator and bind its address to port 9443.
```

Comment:
Kind of self-evident comment that could be dropped IMO. At least drop the port number which is not visible anywhere here and might get out-of-sync with the code easily.

Reply:
That makes a lot of sense. I've removed the reference to port 9443.